projects/plugins/boost/app/assets/src/js/stores/config.ts

```diff
@@ -15,7 +16,15 @@ const dismissedPopOutStore = writable( Jetpack_Boost.dismissedScorePrompts );
 
 export const dismissedPopOuts = {
 	subscribe: dismissedPopOutStore.subscribe,
-	dismiss: ( name: string ) => {
+	dismiss: async ( name: string ) => {
+		await makeAdminAjaxRequest( {
+			action: 'set_show_score_prompt',
+			id: name,
+			value: 'false',
+			// eslint-disable-next-line camelcase
+			nonce: Jetpack_Boost.showScorePromptNonce,
+		} );
+
 		dismissedPopOutStore.update( dismissals => [ ...dismissals, name ] );
```

Comment:
It would give a better perception of speed if we first updated the state and then make the ajax call.

```suggestion
	dismiss: async ( name: string ) => {
		dismissedPopOutStore.update( dismissals => [ ...dismissals, name ] );
		
		await makeAdminAjaxRequest( {
			action: 'set_show_score_prompt',
			id: name,
			value: 'false',
			// eslint-disable-next-line camelcase
			nonce: Jetpack_Boost.showScorePromptNonce,
		} );
```

Reply:
Sure, I'm ok with that. Thanks!